src/libraries/System.Net.Sockets/tests/FunctionalTests/CreateSocketTests.cs

```diff
@@ -122,6 +122,9 @@ public void Ctor_Raw_NotSupported_ExpectedError(AddressFamily addressFamily, Pro
         [InlineData(false, 2)]
         public void CtorAndAccept_SocketNotKeptAliveViaInheritance(bool validateClientOuter, int acceptApiOuter)
         {
+            // 300 ms should be long enough to connect if the socket is actually present & listening.
+            const int ConnectionTimeoutMs = 300;
```

Comment:
I'm wondering if this could go to SocketTestHelper.cs just in case we need to adjust it for ARM runs.

Reply:
Didn't you mean `Configuration.Sockets` actually?
https://github.com/dotnet/runtime/blob/master/src/libraries/Common/tests/System/Net/Configuration.Sockets.cs

There are no configurable items in `SocketTestHelper.cs`.